lib/ex.coffee

```diff
@@ -32,21 +32,19 @@ trySave = (func) ->
 
   deferred.promise
 
-saveAs = (filePath) ->
-  editor = atom.workspace.getActiveTextEditor()
-  fs.writeFileSync(filePath, editor.getText())
```

Comment:
Oh you're right so I guess we should use this saveAs but only when the filename differs from the current filename.